io.vikunja.Vikunja.yml

```diff
@@ -15,18 +15,19 @@ command: vikunja
 
 finish-args:
   - --socket=x11
-  - --talk-name=org.kde.StatusNotifierWatcher
-  - --talk-name=org.freedesktop.Notifications
+  - --share=ipc
   - --env=ZYPAK_SANDBOX_FILENAME=chrome-sandbox
 
 modules:
   - name: vikunja
     buildsystem: simple
     build-commands:
-      - mkdir /app/bin/extra /app/lib/extra
-      - mv *.so* /app/lib/
+      - mkdir ${FLATPAK_DEST}/share/icons/ ${FLATPAK_DEST}/share/applications/ ${FLATPAK_DEST}/app/lib/ ${FLATPAK_DEST}/app/bin/extra/
+      - wget -q https://kolaente.dev/vikunja/desktop/raw/branch/main/build/icon.png -P ${FLATPAK_DEST}/share/icons/vikunja.png
+      - wget -q <will add url> -P ${FLATPAK_DEST}/share/applications/
+      - mv *.so* ${FLATPAK_DEST}/app/lib/
       - install -D vikunja.sh "${FLATPAK_DEST}/bin/vikunja"
-      - mv * /app/bin/extra/
+      - mv * ${FLATPAK_DEST}/app/bin/extra/
```

Comment:
no. that will install in `/app/app` That is not what was asked.

Reply:
Can you show me how to do it the right way?